builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -740,7 +770,7 @@ func resourceVSphereVirtualMachineDelete(d *schema.ResourceData, meta interface{
 	return nil
 }
 
-func waitForNetworkingActive(client *govmomi.Client, datacenter, name string) resource.StateRefreshFunc {
+func waitForNetworkingActive(client *govmomi.Client, datacenter, name string, waitForNetworkInterfaces bool) resource.StateRefreshFunc {
```

Comment:
Instead of doing this would https://github.com/dougm/govmomi/blob/388df2f1d8cb5a0059850fdf10520df7923474ef/object/virtual_machine.go#L252 work?

I would rather use the API to wait for interfaces than maintain our own wait code.